x-pack/plugins/fleet/README.md

```diff
@@ -1,176 +1,140 @@
 # Fleet
 
-## Plugin
+The Fleet plugin allows Elastic Agents to be centrally managed in Kibana.
```

Comment:
(nit) Could we align this with how we describe Fleet in our [public docs](https://www.elastic.co/guide/en/fleet/current/fleet-overview.html#central-management)? 

> Fleet provides a web-based UI in Kibana for centrally managing Elastic Agents and their policies.